src/app/debug/table/table.component.ts

```diff
@@ -283,14 +290,17 @@ export class TableComponent implements OnInit, OnDestroy {
     }
   }
 
-  getStatusColor(metadata: any): string {
-    let statusName = this.viewSettings.currentView.metadataNames.find((name: string) => {
+  getStatusColor(metadata: MetaData): string {
+    let statusName: string | undefined = this.viewSettings.currentView.metadataNames.find((name: string) => {
```

Comment:
```suggestion
    let statusName?: string = this.viewSettings.currentView.metadataNames.find((name: string) => {
```

Reply:
Unfortunately I couldn't get this to work, I also couldn't find anything online about making a variable optional.